src/components/Strategies/Blocks/SigningPopup.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Button, Modal, Progress, Typography } from 'antd';
+import { CheckCircleFilled } from '@ant-design/icons';
+import styled from 'styled-components';
+
+import { calculatePercent } from '../../../utils';
+
+const { Text } = Typography;
+
+interface IProps {
+  visible: boolean;
+  onCancel: () => void;
+  currentTxIndex: number;
+  totalTxsToSign: number;
+}
+function SigningPopup({ visible, onCancel, currentTxIndex, totalTxsToSign }: IProps) {
+  const infoMessage = "Don't worry if you see transactions fail or succeed in Metamask while you are signing!";
```

Comment:
This message would look better if it was just inside the HTML tags instead of being declared here